.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: objective-c
+osx_image: xcode8.3
+
+env:
+  global:
+  - LC_CTYPE=en_US.UTF-8
+  - LANG=en_US.UTF-8
+  - PROJECT=OnDemandPassenger.xcodeproj
+  - SCHEME=OnDemandPassenger
+  - SDK=iphonesimulator10.3
+  - secure: "buEL0pYm46g/5J+p2ZO9o5/fvYsb6Vb4ffagJRTUyfEPi4YnWeQzq7fozYF3ZF9zNKvo455jyO61Do0A617YOoaazPP4lnGzp3Ij/rkcJHIa4q+XC5oEs5NoOETDs/Fd1vZH7+81we2qwDQJDk+o7Y92B/uH3iBpOE8mn2ad4jkDBKS8DU7OuMTmJ60XqqCH58YFWOM+isudvFgnOCFm8LPhUO2yhKUD08ez/k9u0Is6fKDsSyyBtA7CJpZkS33AGKT4mNBsNPxTGRvrtbHEG1Yl0efbrJBT1zaLgcSifvtb15+6K8GVlp9Ykgnuf08nS2jvrAxifmNMyMiIrxnJlosvMDUDMCuy2wR5PaQf4AP+GWHcDDl48bsUqZCzgcMZCZFGZQR7dVPGPYrSlXDUXSIvHOXqwn/woad6j218CQZlmcbhMPyYWwiYeIiFQCJi5REvGnrYpk9ePtN6QZDu/QYr0vjCOTTKhT2ysjNEbEeuAlVk3e13Cwr4Gv/bMKdkkoAVtJpPnvNSqz9D21Xg+9TN+iyxJ/Qa6GGmuha9XVoZvc0NYeWATnQN51732nYuP2vDg7L/BdJMvGn8S0COKYDx5LYllzPrTAiHJT8AbTmFnwFJzTu4i9jNdbJrT2wO3L73UVOsIgLtgvVaeWT/Bfi9oOHbf91WMQjqVXWzmkA="
+  matrix:
+    - DESTINATION="OS=10.3,name=iPhone SE"     SCHEME="$SCHEME"     SDK="$SDK"     RUN_TESTS="NO" POD_LINT="NO"
+    - DESTINATION="OS=9.0,name=iPhone 6"       SCHEME="$SCHEME"     SDK="$SDK"     RUN_TESTS="NO" POD_LINT="NO"
+branches:
+ only:
+ - master
+
+before_install:
+  - rm -rf /Users/travis/Library/Developer/Xcode/DerivedData/OnDemandPassenger-*/
+  - rvm use $RVM_RUBY_VERSION
+  - wget "$URL" -q -O HERE_iOS_SDK_Starter_v3.3.1.tar.gz
+  - tar -xzf HERE_iOS_SDK_Starter_v3.3.1.tar.gz
+  - cp -r HERE_iOS_SDK_Starter/framework/NMAKit.framework .
+
+script: 
+  - set -o pipefail
+  - xcodebuild -version
+  - xcodebuild -showsdks
+
+# Build App in Debug and Run Tests if specified
+  - if [ $RUN_TESTS == "YES" ]; then
+      travis_retry xcodebuild -project "$PROJECT" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES test;
```

Comment:
Why not direct use / set `ENABLE_TESTABILITY` to `YES` / `NO` instead of introducing a `RUN_TEST` variable? That way we could rid of this conditional completely.